install.sh

```diff
@@ -1,8 +1,82 @@
-#!/data/data/com.termux/files/usr/bin/bash
+#!/bin/bash
```

Comment:
Need Support for Termux as well

Reply:
Yes, but you can't have two shebang lines, maybe there's some other way to ensure that this works on Termux as well?